.github/workflows/js-lint.yml

```diff
@@ -0,0 +1,60 @@
+name: JS Code Linting
+
+on:
+  push:
+    branches:
+      - trunk
+    # Only run if JS/JSON/Lint/NVM files changed.
+    paths:
+      - '.github/workflows/js-css-lint-test.yml'
+      - '**.js'
+      - '**.json'
+      - '**.scss'
```

Comment:
Probably not necessary to include `.scss` files here since this is exclusively for JS linting?